next-tavla/src/Shared/hooks/useUpdateLastActive.ts

```diff
@@ -0,0 +1,32 @@
+import { useCallback, useEffect } from 'react'
+import { getBoardSettings } from 'utils/firebase'
+
+function useUpdateLastActive(documentId: string) {
+    const updateLastActive = useCallback(async () => {
+        const settings = await getBoardSettings(documentId)
+        const lastActive = settings.meta?.lastActive ?? 0
+        const lastActiveDate = new Date(lastActive)
+        if (
+            new Date().getTime() - lastActiveDate.getTime() >
+            1000 * 60 * 60 * 24
+        ) {
+            await fetch(`/api/ping`, {
+                method: 'POST',
+                headers: {
+                    'Content-Type': 'application/json',
+                    Authorization: 'documentId ' + documentId,
```

Comment:
Ikke bruk authorization header for dette, bruk heller dynamic paths slik som vi gjør med andre pages: https://nextjs.org/docs/pages/building-your-application/routing/api-routes#dynamic-api-routes

Reply:
Det vil altså være et endepunkt `/ping/[id]`